macro-kroki-api/src/main/java/org/xwiki/contrib/kroki/internal/configuration/KrokiContainerConfiguration.java

```diff
@@ -56,19 +58,21 @@ public class KrokiContainerConfiguration implements KrokiMacroConfiguration
      * @param containerName docker container name
      * @param isContainerReusable false if a new docker container should be instantiated \ even if one with the same
      *     name exists
+     * @param useTls true to use TLS to contact the specified host
      * @param host address to be called for the Kroki service
      * @param port port to be called for the Kroki service
      * @param healthCheckRequestParameters request parameters used to chekc if the Kroki service is online
      * @param configurationSource used to overwrite properties from xwiki.properties file
      */
-    public KrokiContainerConfiguration(String image, String containerName, boolean isContainerReusable, String host,
-        Integer port, HealthCheckRequestParameters healthCheckRequestParameters,
+    public KrokiContainerConfiguration(String image, String containerName, boolean isContainerReusable,
+        boolean useTls, String host, Integer port, HealthCheckRequestParameters healthCheckRequestParameters,
```

Comment:
See my previous comment about using the builder pattern instead of this constructor with all possible parameters.